src/migrations/00043-add-belongs-to-family-dt.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "device type" 
+ADD COLUMN IF NOT EXISTS "belongs to-device family" INTEGER NULL,
+ADD CONSTRAINT "device type_belongs to-device family_fkey" FOREIGN KEY ("belongs to-device family") REFERENCES "device family" ("id");
```

Comment:
I'd expect this to fail because the `device family` table won't exist yet

Reply:
I always forget that the tables need to be created manually :/

Reply:
It's only in the specific case where an existing table needs to be migrated to point to the new table, if it was the new table pointing at an existing table then leaving it to be auto-created would work

Reply:
yeah I know, I meant I forgot that the migrations run before the sbvr tables are created, but makes sense :)